update license (#1083)
telerik/maui-docs: installation/licensing/set-up-your-license.md

* Copy the [downloaded](#downloading-the-license-key) `telerik-license.txt` license key file to your home directory. This makes the license key available to all projects that you develop on your computer:

>     * For Windows: `%AppData%\Roaming\Telerik\telerik-license.txt`. An example of a full path to the `telerik-license.txt` file location: `C:\Users\your-username\AppData\Roaming\Telerik\telerik-license.txt`

Reviewer: ```suggestion
* For Windows: `%AppData%\Roaming\Telerik\telerik-license.txt`. Here's a sample full path to the `telerik-license.txt` file location: `C:\Users\your-username\AppData\Roaming\Telerik\telerik-license.txt`.
```

>     * For Mac/Linux: `~/.telerik/telerik-license.txt`. If `.telerik` folder does not exist, create such, and paste the `telerik-license.txt` file in it.

Reviewer: ```suggestion
* For Mac/Linux: `~/.telerik/telerik-license.txt`. If the `.telerik` folder does not exist, create it and paste the `telerik-license.txt` file in it.
```
